salt/top.sls

```diff
@@ -13,6 +13,7 @@ base:
     - etcd-proxy
     - flannel
     - docker
+    - container-feeder
```

Comment:
We need `container-feeder` in all the machines where `docker` is installed, so I think it is fine as it is.